recipes/pygnome/meta.yaml

```
@@ -0,0 +1,99 @@
+{% set name = "PyGnome" %}
+{% set version = "1.1.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NOAA-ORR-ERD/PyGnome/archive/v{{ version }}.tar.gz
+  sha256: dce3ff20ae28db97379ca57239189a07a9dbef4c79dd662b004a8cd7068c46da
+
+build:
+  skip: True  # [py<39 or py>311 or python_impl == 'pypy']
+  missing_dso_whitelist:
+    - "*/lib_gnome.dll"         # [win]
+    - "*/point_in_polygon.dll"  # [win]
+  number: 0
+  script: cd py_gnome && {{ PYTHON }} -m pip install ./ -vv
+
+requirements:
+  build:
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - numpy                               # [build_platform != target_platform]
+    - netCDF4                             # [build_platform != target_platform]
+    - cython                              # [build_platform != target_platform]
+    - scikit-build-core
```

Same argument as with Cython
```suggestion
    - cython                              # [build_platform != target_platform]
    - scikit-build-core             # [build_platform != target_platform]
```